### PR TITLE
Make it safe to call rungroup actor Interrupt multiple times

### PR DESCRIPTION
**Pull request by RebeccaMahany**
Relates to https://github.com/kolide/launcher/issues/1205

While investigating an autoupdate shutdown issue, noticed that shutdown could block on runner.Interrupt being called multiple times.

### CODE REVIEW

File: ee/desktop/runner/runner.go [hunk 1]
```diff
@@ -222,8 +224,15 @@ func (r *DesktopUsersProcessesRunner) Execute() error {
...
 // It also signals the execute loop to exit, so new desktop processes cease to spawn.
 func (r *DesktopUsersProcessesRunner) Interrupt(_ error) {
-	// Tell the execute loop to stop checking, and exit
-	r.interrupt <- struct{}{}
+	// Non-blocking send to interrupt channel
+	select {
+	case r.interrupt <- struct{}{}:
+		// First time we've received an interrupt, so we've notified r.Execute.
+	default:
+		// Execute loop is no longer running, so there's nothing to interrupt
```
Comment (directionless) [inline ee/desktop/runner/runner.go:232]: Should we log this? Or is it so common it'd just be noise

Comment (RebeccaMahany) [reply ee/desktop/runner/runner.go:232]: It's not common but I'm also not sure it'd be useful?

**Comment by RebeccaMahany** [comment]
Closing in favor of https://github.com/kolide/launcher/pull/1344

**Comment by directionless** [comment]
> Closing in favor of #1344

This one handles more of the actors. Do you think we still need it?

**Pull request closed without merging**